src/Parser/Monk/Windwalker/Modules/Spells/StormEarthAndFire.js

```diff
@@ -36,6 +46,57 @@ class StormEarthAndFire extends Analyzer{
     const reducedCooldownWithTraits = 90 - this.traitsCDReduction;
     return reducedCooldownWithTraits;
   }
+
+  on_toPlayer_applybuff(event) {
+    const spellId = event.ability.guid;
+    if (SPELLS.STORM_EARTH_AND_FIRE_CAST.id === spellId || SPELLS.SERENITY_TALENT.id === spellId) {
+      this.sefCount += 1;
+    }
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+
+    if (!this.combatants.selected.hasBuff(SPELLS.STORM_EARTH_AND_FIRE_CAST.id) && !this.combatants.selected.hasBuff(SPELLS.SERENITY_TALENT.id)) {
+      return;
+    }
+    if (SPELLS.RISING_SUN_KICK.id === spellId) {
```

Comment:
This could be simplified to a switch statement:
```jsx
switch (spellId) {
   case SPELLS.RISING_SUN_KICK.id:
      this.risingSunKicks += 1;
      break;
   case SPELLS.FISTS_OF_FURY_CAST.id:
      this.fistsOfFuries += 1;
      break;
   case SPELLS.STRIKE_OF_THE_WINDLORD.id:
      this.strikeOfTheWindlords += 1;
      break;
   case SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id:
      this.whirlingDragonPunches += 1;
      break;
   default:
      break;
}

```